api/proto/teleport/discoveryconfig/v1/discoveryconfig_service.proto

```diff
@@ -47,6 +47,9 @@ service DiscoveryConfigService {
 
   // DeleteAllDiscoveryConfigs removes all DiscoveryConfigs.
   rpc DeleteAllDiscoveryConfigs(DeleteAllDiscoveryConfigsRequest) returns (google.protobuf.Empty);
+
+  // UpdateDiscoveryConfigState updates an existing DiscoveryConfig resource state object.
+  rpc UpdateDiscoveryConfigState(UpdateDiscoveryConfigStateRequest) returns (DiscoveryConfig);
```

Comment:
Nit: Should this say "Status" instead of "State", for both method name and request struct?

Reply:
Yeah. My brain got confused since I used both but I wanted this to be Update...Status, not state :facepalm: